evennia/contrib/tests.py

```diff
@@ -1283,6 +1283,24 @@ def test_tree_functions(self):
         {'goto': ['menunode_treeselect', {'newindex': 1}], 'key': ['<< Go Back', 'go back', 'back'], 'desc': 'Return to the previous menu.'}]
         self.assertTrue(tree_select.optlist_to_menuoptions(TREE_MENU_TESTSTR, test_optlist, 2, True, True) == optlist_to_menu_expected_result)
 
+# Test field fill
+
+from evennia.contrib import fieldfill
```

Comment:
You have a test-suite with examples already set up in your main module. It seems logical that you trigger this in a unit test and make sure it produces the result you expect. That way you should get good coverage of the components of your contrib.

Reply:
I'll do my best! I may end up needing a bit of help calling and navigating the menu during a test.